Void doesn't package setuptools_scm for Python 2, so remove the dependency.
Leave a placeholder for version number that can be substituted at build.

diff -ur a/setup.cfg b/setup.cfg
--- a/setup.cfg
+++ b/setup.cfg
@@ -35,7 +35,6 @@
 
 [options]
 zip_safe = True
-setup_requires = setuptools_scm
 install_requires = six >= 1.5
 python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*
 packages = find:
diff -ur a/setup.py b/setup.py
--- a/setup.py
+++ b/setup.py
@@ -47,9 +47,7 @@
 
 
 setup(
-      use_scm_version={
-          'write_to': 'dateutil/_version.py',
-      },
+      version=%PKGVERSION%,
       ## Needed since doctest not supported by PyPA.
       long_description = README,
       cmdclass={
